assertk-common/src/main/kotlin/assertk/platform.kt

```diff
@@ -0,0 +1,8 @@
+package assertk
+
+expect
```

Comment:
the `expect` modified before the KDoc seems like an odd style choice

Reply:
Yes, I don't like it either :/ but that's the only way to make detekt happy :(
It worth a but for detekt I think.

Reply:
I think it's worth removing detekt at this point. It seems to be creating more friction than helping @evant